requirements.txt

```diff
@@ -24,6 +24,8 @@ pyramid_mako
 pyramid_tm
 python-bugzilla
 python-fedora
+python-hawkey
```

Comment:
It looks like hawkey is not available under PyPI (or at least, not under this name). This is OK, as there are other Bodhi dependencies that are also not on PyPI (like Koji), but it does cause the pip container to fail to build:

```
pip	Collecting python-hawkey (from -r /bodhi/requirements.txt (line 27))
pip	[91m  Could not find a version that satisfies the requirement python-hawkey (from -r /bodhi/requirements.txt (line 27)) (from versions: )
pip	[0m[91mNo matching distribution found for python-hawkey (from -r /bodhi/requirements.txt (line 27))
```

To fix this, you can add it to ```devel/ci/pip-packages``` so that it gets installed via RPM for that container.